core/src/main/java/tc/oc/pgm/controlpoint/ControlPointParser.java

```diff
@@ -130,6 +132,12 @@ public static ControlPointDefinition parseControlPoint(
             "capture rule",
             ControlPointDefinition.CaptureCondition.EXCLUSIVE);
 
+    if (attrContested != null
+        && captureCondition != ControlPointDefinition.CaptureCondition.EXCLUSIVE) {
+      throw new InvalidXMLException(
+          "Contested rate may only apply to exclusive capture rule.", attrContested);
+    }
```

Comment:
Why is this a limitation? I see it has been implemented in a way that would not support anything except exclusive, however, i see no reason why it should be this way.
On both MAJORITY and LEAD there are cases where contesting can occur, some examples:
 - MAJORITY or LEAD:  Team A has 2 players, team B has 2 players
 - MAJORITY: Team A has 5 players, Team B has 3 players, team C has 2 players